docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '2'
+services:
+  api:
+    build: .
+    image: businesses-api
```

Comment:
We should use a more relevant name, like `courses-api`, as this project has no context regarding businesses.